Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedCursorAndChangeStreamOperations.swift

```diff
@@ -17,10 +17,9 @@ struct CreateChangeStream: UnifiedOperationProtocol {
     }
 
     static var knownArguments: Set<String> {
-        Set(
-            CodingKeys.allCases.map { $0.rawValue } +
-                ChangeStreamOptions().propertyNames
-        )
+        // TODO: SWIFT-1385 Remove `comment` and integrate into ChangeStreamOptions
+        Set(Set(CodingKeys.allCases.map { $0.rawValue }).union(
```

Comment:
This outermost `Set` isn't required, since the return value of `.union` is a `Set` itself. I think the `"comment"` one can just be `["comment"]` due to type inference also.